packages/rainbowkit/src/wallets/walletConnectors/kaikasWallet/kaikasWallet.ts

```diff
@@ -17,8 +18,8 @@ export const kaikasWallet = ({
 
   const shouldUseWalletConnect = !isKaikasWalletInjected;
 
-  const getUri = (uri: string) => {
-    return `kaikas://walletconnect?uri=${encodeURIComponent(uri)}`;
+  const getUri = () => {
+    return `kaikas://wallet/browser?url=${encodeURI(window.location.origin)}`
```

Comment:
@DanielSinclair Currently kaikas do not seem to be being managed. kaikas is a wallet mainly used on the Klaytn mainnet, and it was said that the Klaytn mainnet and finschia mainnet will be integrated in the third quarter of this year. I may update it again at that time, but currently it is not well supported, so it would be best if it works in the browser.

https://medium.com/lineblockchain/en-say-hello-to-kaia-1b1d489a8b8f